source/frontend/src/models/generate/GenerateAddress.ts

```diff
@@ -43,8 +43,9 @@ export class Api_GenerateAddress {
       .filter(a => !!a)
       .join(', ');
 
-    this.address_String_multiline_compressed =
```

Comment:
@eddherrera is this used in any other templates? If so does this require us to update those templates?

Reply:
no, this is the only template so far that uses this format. I created this variation of the address so it's new.